src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,115 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static int CAPACITY = 10;
+    private static double SIZE_OF_INCREASE = 1.5;
```

Comment:
SIZE_OF_INCREASE should be a constant (final) and its name should follow the naming conventions for constants, using uppercase letters and underscores.